### DISCUSSION TITLE
Please bring back the “Generate” button for slugs (or a replacement)

### DISCUSSION THREAD

**Discussion by coreyward** [sanity-io/sanity#3440] [Sanity Studio Feedback]
I'm surprised there is no documentation on how to treat slugs like in v2 where a button adjacent to the input can be used to generate the slug programmatically. Is there an expectation that this is something to use a custom input for?

**Comment by kmelve** [comment]
This is a bug, and not an intentional breaking change. [The code](https://github.com/sanity-io/sanity/blob/next/packages/%40sanity/form-builder/src/inputs/Slug/SlugInput.tsx) for it is still there, but for some reason, it doesn't run. We'll get to it!

**Comment by rexxars** [comment]
Fixed in [v3.0.0-dev-preview.12](https://github.com/sanity-io/sanity/releases/tag/v3.0.0-dev-preview.12) - thanks for reporting!
